prover/src/pp.rs

```diff
@@ -121,3 +149,28 @@ where
     show_pp(&pp);
     Ok(pp)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::srs::test_srs::gen_test_srs_to_file;
+
+    // This test will not pass unless we use the full 26-variable SRS
+    #[test]
+    #[ignore]
+    fn test_gen_pp_with_srs() {
+        gen_to_file(1, true, true, "test_pp.zst", Some("../test_srs.zst")).unwrap();
+    }
+
+    fn test_srs_gen() {
+        gen_test_srs_to_file(10, "small_test_srs.zst").unwrap();
+        let _srs: SRS = load_srs("small_test_srs.zst").unwrap();
+    }
+
+    #[test]
+    fn test_load_srs() {
+        test_srs_gen();
+        let srs: SRS = load_srs("small_test_srs.zst").unwrap();
+        println!("Loaded SRS for {} variables", srs.max_num_vars);
+    }
```

Comment:
tests should not leave any files laying around

Reply:
is there a way to delete the file here? we can just '#[ignore]' this test.

Reply:
You shouldn't have any leftover even after running `cargo test --ignored`
One way is to make function generic over something that's `: Read` and test it with `& [u8]`, something of a mock interface.

As an example, [tools tests](https://github.com/nexus-xyz/nexus-zkvm/blob/d07d60698273a86be81b6ab76e66119b9d5347c1/tools/tools-dev/src/command/dev/config.rs#L143-L173) just use strings.

Reply:
hm, i'm not really sure how to change the functions here to work that way, since the files are going to be very big. I'm just going to take these tests out for now, since the compression-cli PR will let us test this functionality from the main interface.